packages/connect-popup/src/view/selectDevice.ts

```diff
@@ -123,12 +122,16 @@ export const selectDevice = (payload: UiRequestSelectDevice['payload']) => {
 
             // handle unreadable device
             if (device.type === 'unreadable') {
-                const os = getOS();
+                const { systemInfo } = getState();
                 // default explanation: contact support
 
                 let explanationContent = `Please <a href="${TREZOR_SUPPORT_URL}" target="_blank" rel="noreferrer noopener" onclick="window.closeWindow();">contact support.</a>`;
                 // linux + LIBUSB_ERROR handling
-                if (os === 'linux' && device.error.indexOf(ERRORS.LIBUSB_ERROR_MESSAGE) >= 0) {
+                if (
+                    systemInfo?.os?.name &&
+                    systemInfo.os.name.toLocaleLowerCase() === 'linux' &&
```

Comment:
this does not play very well. there could be Ubuntu, Mint, Linux...

Reply:
maybe like this? 830a81d5615e10d7694939925a66dcf542905583 fae721e856a2dcb0863fd47e45ff5ffc28ada46c
or maybe I could use ['rpm32','rpm64', 'deb32', 'deb64'].includes(systemInfo.platform)